sirepo/package_data/static/html/http-not-found.html

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-2">
       <h1>Not Found</h1>
-      The requested URL was not found on this server.
+      <p>The requested URL was not found on this server.</p>
```

Comment:
is not found? too-large.html was changed for was to is.

Reply:
This is for 404.